src/MB_Toolbox.php

```diff
@@ -182,6 +182,40 @@ public function createDrupalUser($user) {
   }
 
   /**
+   * Get the users password reset link via Drupal end point.
+   * https://github.com/DoSomething/dosomething/wiki/API#get-password-reset-url
+   *
+   * POST https://beta.dosomething.org/api/v1/users/[uid]/password_reset_url
+   *
+   * @param int $uid
+   *    UID of the user were getting the reset URL for.
+   *
+   * @return string $passwordResetUrl
+   *    The string supplied by the Drupal endpoint /password_reset_url or NULL on failure
+   */
+   public function getPasswordResetURL($uid) {
+     $curlUrl = $this->settings['ds_drupal_api_host'];
+     $port = $this->settings['ds_drupal_api_port'];
+     if ($port > 0 && is_numeric($port)) {
+       $curlUrl .= ':' . (int) $port;
+     }
+     $curlUrl .= self::DRUPAL_API . '/users' . $uid . '/password_reset_url';
```

Comment:
Based on https://github.com/DoSomething/dosomething/wiki/API#get-password-reset-url it looks like you're missing a "/" infront of `$uid`.